shotover/benches/benches/codec/cassandra.rs

```diff
@@ -30,10 +30,93 @@ fn criterion_benchmark(c: &mut Criterion) {
             },
         }))];
 
-        let (_, mut encoder) =
+        let (mut decoder, mut encoder) =
             CassandraCodecBuilder::new(Direction::Sink, "cassandra".to_owned()).build();
 
-        group.bench_function("encode_system.local_query", |b| {
+        encoder.set_startup_state_ext("NONE".to_string(), Version::V4);
+
+        group.bench_function("encode_system.local_query_v4_no_compression", |b| {
+            b.iter_batched(
+                || messages.clone(),
+                |messages| {
+                    let mut bytes = BytesMut::new();
+                    encoder.encode(messages, &mut bytes).unwrap();
+                    black_box(bytes)
```

Comment:
This line could be just `bytes` as criterion will automatically put the return value through a black_box.